pkg/manifest/os.go

```diff
@@ -287,6 +287,21 @@ func (p *OS) getContainerSources() []container.SourceSpec {
 	return p.OSCustomizations.Containers
 }
 
+func tomlPkgsFor(distro Distro) []string {
+	switch distro {
+	case DISTRO_EL7:
+		// Supported by upstream but superseeded by tomllib in stdlib
+		return []string{"python3-toml"}
```

Comment:
This is a slightly weird one - I added it because it was caught by the old codes "default" but I'm not actually sure this works.

Reply:
We never covered the EL7 case with toml explicitly because there's no scenario where it can be needed.  None of the image types we build for RHEL 7 will ever need to write or read toml (`org.osbuild.containers.storage.conf` is only ever used in ostree-based images and `org.osbuild.bootc.install.config` is very new), and we don't support running osbuild on RHEL 7 either.

I checked to make sure and there's no `python3-toml` available for EL7 rel-eng, which is what we use to get python3 and its modules.  It does have `python-pytoml` (python2) in its main repos, which of course is no good to us.

I think in this case it makes more sense to panic (with a note that it should be an error, once we get around to replacing all panics with errors) if any image definition tries to use toml in EL7.

Reply:
I updated this now to reflect reality (hopefully) a bit better. Still need to see if I can write a small test but it should have the right packages and comments now.